pom.xml

```diff
@@ -46,8 +46,8 @@
         <graalvmHome>${env.GRAALVM_HOME}</graalvmHome>
         <postgres.url>jdbc:postgresql:hibernate_orm_test</postgres.url>
 
-        <quarkus.version>0.26.0</quarkus.version>
-        <camel-quarkus.version>0.2.1-SNAPSHOT</camel-quarkus.version>
+        <quarkus.version>0.27.0</quarkus.version>
+        <camel-quarkus.version>0.3.1-SNAPSHOT</camel-quarkus.version>
```

Comment:
This should be `0.3.1`?

Reply:
It has not yet been released, will update the pr when camel-quarkus lands in the staging repo